showkase-screenshot-testing/src/main/java/com/airbnb/android/showkase/screenshot/testing/ShowkaseScreenshotTest.kt

```diff
@@ -80,6 +87,8 @@ interface ShowkaseScreenshotTest {
         name: String,
         group: String,
         styleName: String? = null,
+        tags: List<String> = emptyList(),
```

Comment:
@vinaygaba Are you okay with adding the tags and metadata here as well? We need this for correct attribution of our screenshots since we will likely have duplicate group/name/styleName and are using the `extraMetadata` to handle additional categorization.

This would also be a breaking change.